files/__addonLocation__/package.json

```diff
@@ -29,12 +30,35 @@
   },
   "devDependencies": {
     "@babel/core": "^7.17.0",
-    "@babel/eslint-parser": "^7.18.2",
+    <% if (typescript) { %>"@babel/preset-typescript": "7.18.6"<% } else { %>"@babel/eslint-parser": "^7.18.2"<% } %>,
     "@babel/plugin-proposal-class-properties": "^7.16.7",
     "@babel/plugin-proposal-decorators": "^7.17.0",
     "@babel/plugin-syntax-decorators": "^7.17.0",
-    "@embroider/addon-dev": "^1.0.0",
-    "@rollup/plugin-babel": "^5.3.0",
+    "@embroider/addon-dev": "^2.0.0",<% if (typescript) { %>
+    "@glint/core": "^0.9.1",
+    "@glint/environment-ember-loose": "^0.9.1",
```

Comment:
Well, I don't really think this makes it into "prime time". 😅 At least until this blueprint replaces the official v1 addon blueprint *eventually*. So as this blueprint is still more on the experimental side of things, I wouldn't mind this to use other more experimental things (pre-stable Glint). 🤷

Reply:
Haha that's fair—it's certainly true that anyone using the v2 blueprint today knows they're opting into something that isn't guaranteed to be 100% baked yet.

This will just be the first time (that I know of!) that anyone will find themselves set up with Glint without having gone out of their way to read the docs and decide it's something they want. This blueprint feels to me like a reasonable place to take that step, but I did want to call it out and make sure Chris and James are on the same page